setup.py

```diff
@@ -21,8 +21,7 @@
 #
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
-NAME = "opal"
-VERSION = "1.0.0"
```

Comment:
why are we removing VERSION from setup

Reply:
we're adding scm_version so we use the git tags for the version